service/prometheus/group.go

```diff
@@ -14,10 +14,6 @@ func FilterInvalidServices(services []v1.Service) []v1.Service {
 			continue
 		}
 
-		if _, ok := service.ObjectMeta.Annotations[CertificateAnnotation]; !ok {
```

Comment:
We can also remove this annotation entirely. We assume the cluster annotation value matches the secret `clusterID` label.